web/packages/teleterm/src/mainProcess/windowsManager.ts

```diff
@@ -385,6 +456,31 @@ export class WindowsManager {
       ...getPositionAndSize(),
     };
   }
+
+  private async confirmIfShouldRunInTrayOnce(): Promise<boolean> {
+    const runInBackgroundState = this.fileStorage.get(
+      'runInBackground'
+    ) as RunInBackgroundState;
+    if (runInBackgroundState?.notified) {
+      return true;
+    }
+
+    const isMac = this.settings.platform === 'darwin';
+
+    const { response } = await dialog.showMessageBox(this.window, {
+      type: 'info',
+      message: isMac
+        ? 'Teleport Connect will continue running in the menu bar'
+        : 'Teleport Connect will continue running in the system tray',
+      detail:
+        'VNet and connections to database, kube clusters and apps will remain active.',
+      buttons: ["Don't run in background and quit", 'OK'],
+      defaultId: 1,
+    });
+    const state: RunInBackgroundState = { notified: true };
+    this.fileStorage.put('runInBackground', state);
+    return response !== 0; // true if 'OK'
+  }
```

Comment:
is there a way to change this after i've selected the option once?

Reply:
Yes, you can change it in the app config.